sklearn/metrics/tests/test_ranking.py

```diff
@@ -73,16 +73,16 @@ def make_prediction(dataset=None, binary=False):
 
     # run classifier, get class probabilities and label predictions
     clf = svm.SVC(kernel='linear', probability=True, random_state=0)
-    probas_pred = clf.fit(X[:half], y[:half]).predict_proba(X[half:])
+    y_score = clf.fit(X[:half], y[:half]).predict_proba(X[half:])
```

Comment:
Could you revert those back, too?

Reply:
The name `probas_pred` appears only as the name of a positional argument of the function `precision_recall_curve`. All the other functions use `y_score` instead.

Reply:
The point is more: You call `predict_proba` and assign it to `probas_pred`. Why rename it to `y_score`?

Reply:
To align it with the input parameter name for:
- roc_curve(y_true, y_score
- roc_auc_score(y_true, y_score
- average_precision_score(y_true, y_score
- _average_precision(y_true, y_score)
- _average_precision_slow(y_true, y_score)

besides:
- precision_recall_curve(y_true, probas_pred

The name `probas_pred` is only used as input parameter name for precision_recall_curve. That is the reason I proposed renaming this parameter to align it with all the rest. It is too complicated the renaming. OK. But it makes no sense calling probas_pred to the input of all other functions, just because of this only function.

Reply:
Thanks.